<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExportDialog</class>
 <widget class="QDialog" name="ExportDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>574</width>
    <height>615</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout">
   <item>
    <layout class="QVBoxLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>&lt;b&gt;Predefined exports&lt;/b&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTreeView" name="uiPredefined">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <item>
      <widget class="QPushButton" name="pushRemoveExport">
       <property name="text">
        <string>Re&amp;move Export</string>
       </property>
       <property name="icon">
        <iconset resource="common.qrc">
         <normaloff>:/images/delete.png</normaloff>:/images/delete.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <item>
      <layout class="QVBoxLayout">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>&lt;b&gt;All fields&lt;/b&gt;</string>
         </property>
         <property name="buddy">
          <cstring>uiAllFields</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTreeView" name="uiAllFields">
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::ExtendedSelection</enum>
         </property>
         <property name="sortingEnabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout">
       <item>
        <spacer>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushAdd">
         <property name="text">
          <string>A&amp;dd</string>
         </property>
         <property name="icon">
          <iconset resource="common.qrc">
           <normaloff>:/images/next.png</normaloff>:/images/next.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushRemove">
         <property name="text">
          <string>&amp;Remove</string>
         </property>
         <property name="icon">
          <iconset resource="common.qrc">
           <normaloff>:/images/previous.png</normaloff>:/images/previous.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushRemoveAll">
         <property name="text">
          <string>Remo&amp;ve all</string>
         </property>
         <property name="icon">
          <iconset resource="common.qrc">
           <normaloff>:/images/trash.png</normaloff>:/images/trash.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushSave">
         <property name="text">
          <string>&amp;Save list</string>
         </property>
         <property name="icon">
          <iconset resource="common.qrc">
           <normaloff>:/images/save.png</normaloff>:/images/save.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>&lt;b&gt;Fields to export&lt;/b&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListView" name="uiSelectedFields">
         <property name="dragEnabled">
          <bool>true</bool>
         </property>
         <property name="dragDropMode">
          <enum>QAbstractItemView::InternalMove</enum>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::ExtendedSelection</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>&lt;b&gt;Options&lt;/b&gt;</string>
       </property>
       <property name="buddy">
        <cstring>uiFormat</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QComboBox" name="uiFormat"/>
     </item>
     <item row="1" column="1">
      <widget class="QCheckBox" name="uiAddFieldNames">
       <property name="text">
        <string>Add &amp;field names</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="2">
      <widget class="QCheckBox" name="uiImportCompatible">
       <property name="toolTip">
        <string>&lt;p&gt;
Check this field if you want to be able to import this information back to OpenERP.
&lt;/p&gt;

&lt;p&gt;
Note that you may also be interested in adding ID or Database ID fields, so OpenERP can match CSV and database's records when you import it later.
&lt;/p&gt;
&lt;p&gt;
On the other hand, import incompatible files will show some information in a way users can better understand it and it's probably what you expect if you do not intend to import data back to OpenERP.
&lt;/p&gt;</string>
       </property>
       <property name="text">
        <string>Import compatible</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushAccept">
       <property name="text">
        <string>&amp;Accept</string>
       </property>
       <property name="icon">
        <iconset resource="common.qrc">
         <normaloff>:/images/ok.png</normaloff>:/images/ok.png</iconset>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushCancel">
       <property name="text">
        <string>&amp;Cancel</string>
       </property>
       <property name="icon">
        <iconset resource="common.qrc">
         <normaloff>:/images/cancel.png</normaloff>:/images/cancel.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="common.qrc"/>
 </resources>
 <connections/>
</ui>
